examples/game/lib/action_spline.dart

```diff
@@ -24,11 +24,11 @@ class ActionSpline extends ActionInterval {
 
   final Function setter;
   final List<Point> points;
+  double tension = 0.5;
```

Comment:
Ok, I might reconsider how this is set to make it more consistent! I.e. not having some properties editable and other final.